spaces/S000122/README.md

```diff
@@ -6,13 +6,13 @@ refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
 ---
-Let $\mathbb{Z}^+$ denote the set of positive integers, $Y = (\mathbb{Z}^+)^{<\omega}$ be the set of all finite sequences of positive integers, $W = \{A \subset Y\ |\ |A| = 2\}$. Gustin's Sequence Space is the set $X = Y \cup (\mathbb{Z}^+ \times W)$ with topology defined as follows:
+Let $\mathbb{Z}^+$ denote the set of positive integers, $Y$ be the set of all finite sequences of positive integers of even length, and $W = \{A \subseteq Y\ |\ |A| = 2\}$. Gustin's Sequence Space is the set $X = Y \cup (\mathbb{Z}^+ \times W)$ with topology defined as follows:
 
-For any $\alpha, \beta \in Y$, let $\alpha \cap \beta$ be the sequence formed by adjoining $\beta$ to the end of $\alpha$. Let $\alpha \geq i \in \mathbb{Z}$ abbreviate $a \geq i$ for every $a \in \alpha$. Let $\beta \supset_i \alpha$ abbreviate $\exists \gamma \geq i$ with $\beta = \alpha\gamma$ and $U_i(\alpha) = \{\beta \in Y\ |\ \beta \supset_i \alpha\}$.
+For any $\alpha, \beta \in Y$, let $\alpha\beta$ be the sequence formed by adjoining $\beta$ to the end of $\alpha$. Let $\alpha \geq i \in \mathbb{Z}$ abbreviate $a \geq i$ for every $a \in \alpha$. Let $\beta \supseteq_i \alpha$ abbreviate $\exists \gamma \geq i$ with $\beta = \alpha\gamma$ and $U_i(\alpha) = \{\beta \in Y\ |\ \beta \supseteq_i \alpha\}$.
```

Comment:
```suggestion
For any finite sequences $\alpha, \beta$ of arbitrary length, let $\alpha\beta$ be the sequence formed by adjoining $\beta$ to the end of $\alpha$. Let $\alpha \geq i \in \mathbb{Z}$ abbreviate $a \geq i$ for every $a \in \alpha$. Let $\beta \supseteq_i \alpha$ abbreviate $\exists \gamma \geq i$ with $\beta = \alpha\gamma$ and $U_i(\alpha) = \{\beta \in Y\ |\ \beta \supseteq_i \alpha\}$.
```

Reply:
I think this is better because it clarifies that the definitions are for arbitrary finite sequences, and not just when $\alpha, \beta$ are of even length.